### PR TITLE
Fix/57 support for additionals fields for query path params

### LINKED ISSUE #57: Support for description/example/etc... from schema in query/path parameter
Hapic should create query/path parameter with description and example when available.

### PR DESCRIPTION
**Pull request by inkhey**
closes #57 ?

### CODE REVIEW

File: hapic/doc.py [hunk 1]
```diff
@@ -33,6 +68,63 @@ def generate_schema_ref(spec:APISpec, schema: marshmallow.Schema) -> str:
...
+        if 'description' not in description:
+            description['description'] = ""
+        description['description'] = '{description}\n\n*example value: {example}*'.format(  # nopep8
+            description=description['description'],
+            example=schema['example']
+        )
+    return description
+
+
 def bottle_generate_operations(
```
Comment (lebouquetin) [inline hapic/doc.py]: Actually, why do we have a bottle related function in the `hapic/doc.py` module (which should include generic stuff only) ?

File: hapic/__init__.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from hapic.data import HapicData
 from hapic.infos import __version__
 
+__version__ = '0.41'
```
[the remark this answers is not in the document]
Comment (inkhey) [reply hapic/__init__.py]: Yes that's an error. __version__ should not be declared in __init__.py, it is already imported here.

File: hapic/doc.py (hunk 1 above)
Comment (inkhey) [reply hapic/doc.py]: legacy name of this method, Must be modify .

File: additionals_fields.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+## Addtionals fields supported
+
+Using marshmallow schema, you can add some extra field in field who are
+automatically added into field description in schema.
+```
```
[the remark this answers is not in the document]
Comment (inkhey) [reply additionals_fields.md]: Done.

File: hapic/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (inkhey) [reply hapic/__init__.py]: Done.

File: hapic/doc.py (hunk 1 above)
Comment (inkhey) [reply hapic/doc.py]: see  #64

**Pull request merged**